packages/plugin-did-you-mean/src/commands/config/autocorrect/status.test.ts

```diff
@@ -0,0 +1,50 @@
+import AutocorrectStatus from './status.js'
+import {isAutocorrectEnabled} from '../../../services/conf.js'
+import {Config} from '@oclif/core'
+import {describe, expect, it, vi} from 'vitest'
+import {mockAndCaptureOutput} from '@shopify/cli-kit/node/testing/output'
+
+vi.mock('../../../services/conf.js')
+
+describe('Autocorrect Status', () => {
+  it('Displays autocorrect off message if Autocorrect is off', async () => {
+    // Given
+    vi.mocked(isAutocorrectEnabled).mockReturnValue(false)
+    const config = new Config({root: __dirname})
+    const outputMock = mockAndCaptureOutput()
+
+    // When
+    await new AutocorrectStatus([], config).run()
+
+    // Then
+    expect(outputMock.info()).toMatchInlineSnapshot(`
+    "╭─ info ───────────────────────────────────────────────────────────────────────╮
+    │                                                                              │
+    │  Autocorrect off. You'll need to confirm corrections for mistyped commands.  │
+    │                                                                              │
+    ╰──────────────────────────────────────────────────────────────────────────────╯
+    "
+  `)
+  })
+
+  it('Displays autocorrect on message if Autocorrect is on', async () => {
+    // Given
+    vi.mocked(isAutocorrectEnabled).mockReturnValue(true)
```

Comment:
I have the feeling there is a better way to mock the returned value. Right now each test passes in isolation, but the test suite fails for this file

Reply:
This is the correct way of mocking stuff 🤔 it should work, I'll pull the branch and try myself

Reply:
Figured out we have to clear the captured output between tests